SpatialGDK/Source/SpatialGDK/Public/Interop/GlobalStateManager.h

```diff
@@ -89,7 +87,9 @@ class SPATIALGDK_API UGlobalStateManager : public UObject
 	uint32 SchemaHash;
 
 	// Startup Actor Manager Component
+	bool bHasSentReadyToBegin;
```

Comment:
This is slightly ambiguous, just reading the header I assume this means non-auth servers have suggested they are ready to play, but could mean the gsm worker has sent the `bCanBeginPlay` update.